src/plugins/pcr_primer_design_for_dna_assembly/src/tasks/ExtractPrimerTask.cpp

```diff
@@ -89,7 +135,7 @@ void ExtractPrimerTask::run() {
                 backboneStrand = U2Strand(U2Strand::Complementary);
             }
             productSequence.seq.prepend(settings.backboneSequence);
-            fragmentRegion.startPos =+ settings.backboneSequence.length();
+            fragmentRegion.startPos = +settings.backboneSequence.length();
```

Comment:
question: why do you have '+' here?

Reply:
We need to set new start position for product because we have backbone sequence between primer and product.